static/src/scripts/app/mixens/BaseCollectionMixen.coffee

```diff
@@ -23,4 +23,9 @@ define [
       _.each keys, (key) =>
         uri.addQueryParam(key, @urlParams[key])
 
-      uri.toString()
\ No newline at end of file
+      uri.toString()
+
+    sync: ->
+      # do not remove
```

Comment:
Great comment Kevin. Good job. Fantastic reason given there. It's about the Mixens `SyncingMixen` and `AuthMixen` implementing `sync`. We don't want to break the chain.